CHANGELOG/CHANGELOG-1.4.md

```diff
@@ -15,6 +15,8 @@ When cutting a new release, update the `unreleased` heading to the tag being gen
 
 ## unreleased
 
+* [ENHANCEMENT] [#773](https://github.com/k8ssandra/k8ssandra-operator/issues/773) Allow for CORS to be enabled on Reaper
```

Comment:
The changelog entry needs to go to `CHANGELOG-1.5.md` now.
We're only pushing bug fixes to the 1.4 branch ;)

Reply:
@jdonenine, still waiting on this change until I can merge the PR 😉

Reply:
My bad, I never saw this, i'll take care of it tonight.